src/scripts/runscripts.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
```

Comment:
Is there any reason why you are using `bash` instead of `python` script?

Reply:
First I ran the generated file from terminal, when i tested my work. Then when I had more files, had the idea to write the commands in a script. This is why I'm using bash. It looked good idea.

Reply:
Since you are about to use `python` scripts in this repository, I suggest to rewrite this as well.